develop/CICD/Skyline Communications/Publish to Catalog/Publish_To_Catalog.md

```diff
@@ -249,3 +249,13 @@ As dashboards are a purely visual artifact, the source code is rarely changed di
          </Dashboard>
       </Dashboards>
       ```
+
+## Publishing Visio files
+
+Visio files can be used as a custom user interface for elements, services, or views. They are generally published as single .vsix files.
```

Comment:
Just to be sure: is it correct that they are published as .vsix files? I'd expect .vsdx instead...

Reply:
You're right, it should be .vsdx. I wrote a typo.